compiler/cmd/src/compile_bpf.rs

```diff
@@ -250,6 +255,47 @@ mod test {
         println!("{}", eunomia_include);
     }
 
+    #[test]
+    fn test_generate_custom_btf() {
+        let test_bpf = include_str!("../test/client.bpf.c");
+        let test_event = include_str!("../test/event.h");
+        let tmp_dir = path::Path::new(TEMP_EUNOMIA_DIR);
+        let tmp_dir = tmp_dir.join("test_compile_bpf");
+        fs::create_dir_all(&tmp_dir).unwrap();
+        fs::write(
+            tmp_dir.join("other_header.h"),
+            include_str!("../test/other_header.h"),
+        )
+        .unwrap();
+        let source_path = tmp_dir.join("client.bpf.c");
+        println!("source_path: {}", source_path.to_str().unwrap());
+        fs::write(&source_path, test_bpf).unwrap();
+        let event_path = tmp_dir.join("event.h");
+        fs::write(&event_path, test_event).unwrap();
+        let tmp_workspace = TempDir::new().unwrap();
+        init_eunomia_workspace(&tmp_workspace).unwrap();
+
+        let mut args = Options {
+            tmpdir: tmp_workspace,
+            compile_opts: CompileOptions {
+                btfgen: true,
```

Comment:
This option takes up a lot of time and resources in unpacking and generating custom btf files.
Do we really need this test and could it affect development when it takes up too much time?